src/SearchResults.js

```diff
@@ -46,9 +47,14 @@ export default function Search({
   return (
     <SoundList
       header={
-        showHeader
-          ? `Search results for "${searchQuery || searchValue}"`
-          : undefined
+        showHeader && (
```

Comment:
the `header` prop expects a string. The `Helmet` component needs to be adjacent to `SoundList` like in this example:

```
<>
  <SoundList ... />
  <Helmet ... />
</>
```

Reply:
Hold on, how can the` <Helmet>` prob be adjacent to `<Soundlist>` if it's inside of `<Soundlist>`? @amilajack